src/main/java/data/JsonConverter.java

```diff
@@ -48,18 +52,27 @@ public File createFile(String vendorID) {
     return file;
   }
 
-  public boolean updateFile(Vendor vendor) {
-    createFile(vendor.getVendorID());
+  public boolean updateFile(Vendor vendor, boolean isTesting) {
```

Comment:
We really want to avoid having a flag like isTesting in our APIs. That gets ugly and hard to maintain really quickly. Changing the base path approach will remove the need for this